django/contrib/contenttypes/fields.py

```diff
@@ -236,21 +218,29 @@ def __get__(self, instance, cls=None):
         if instance is None:
             return self
 
+        # Make sure to use ContentType.objects.get_for_id() to ensure that
```

Comment:
Okay, perhaps it could be simplified slightly to something like: `Use ContentType.objects.get_for_id() so that lookups are cached...`